vizro-core/examples/visual-vocabulary/pages/_pages_utils.py

```diff
@@ -11,13 +11,37 @@
 # To disable logging info messages caused by black.format_str: https://github.com/psf/black/issues/2058
 logging.getLogger("blib2to3").setLevel(logging.ERROR)
 
+VIZRO_CODE_TEMPLATE = """
+import vizro.models as vm
+from vizro import Vizro
 
-def make_code_clipboard_from_py_file(filepath: str):
+{example_code}
+
+page = vm.Page(title="{title}", components=[vm.Graph(figure=fig)])
+dashboard = vm.Dashboard(pages=[page])
+Vizro().build(dashboard).run()
+"""
+# TODO HERE: isort? Do like vizro-ai.
+# Make sure unused imports like vizro.models.types.capture disappear
+# Roll out changes across all files - need to modify all example files other than magnitude ones and import fig directly
+# from those files in all vm.Graph().
+
+
+def make_code_clipboard_from_py_file(filepath: str, mode="vizro"):
     # Black doesn't yet have a Python API, so format_str might not work at some point in the future.
     # https://black.readthedocs.io/en/stable/faq.html#does-black-have-an-api
-    filepath = Path(__file__).parents[1] / "pages/examples" / filepath
+    example_code = (Path(__file__).parents[1] / "pages/examples" / filepath).read_text()
+
+    if mode == "vizro":
+        example_code = VIZRO_CODE_TEMPLATE.format(title="Title", example_code=example_code)
+    else:
+        replacements = {"import vizro.plotly.express as px": "import plotly.express as px", '@capture("graph")': ""}
```

Comment:
```suggestion
        replacements = {
            'import vizro.plotly.express as px': 'import plotly.express as px', 
            'from vizro.models.types import capture': '',    
            '@capture("graph")': '',
        }
```

Reply:
This is done automatically using `autoflake` in [9a8e3c7](https://github.com/mckinsey/vizro/pull/824/commits/9a8e3c7c6a42fbfbbfedf1e77da38d1e95cd608d).